iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -382,16 +382,17 @@ void RouDi::processMessage(const runtime::IpcMessage& message,
     }
     case runtime::IpcMessageType::FIND_SERVICE:
     {
-        if (message.getNumberOfElements() != 3)
+        if (message.getNumberOfElements() != 4)
```

Comment:
oh boy, we need a proper RPC handling for RouDi :smile:

Reply:
Yes, and also isolate the message processing logic for all specific commands,

Reply:
@MatthiasKillat right. I also have a proper solution for this ready in my mind since ages. Just need to find some time.